webapp/src/main/webapp/config/listViewConfig-authorInAuthorship.xml

```diff
@@ -10,75 +10,101 @@
-            OPTIONAL { ?infoResource bibo:pageStart ?startPage }
-            OPTIONAL { ?infoResource bibo:pageEnd ?endPage }
-            OPTIONAL { ?infoResource bibo:doi ?doi }
-            OPTIONAL { ?infoResource bibo:pmid ?doi }
-            OPTIONAL { ?infoResource bibo:isbn10 ?isbn10 }
-            OPTIONAL { ?infoResource bibo:isbn13 ?isbn13 }
-            OPTIONAL { ?infoResource core:placeOfPublication ?locale }
+            OPTIONAL { ?infoResource vitro:mostSpecificType ?irsubclass }
+            OPTIONAL { ?infoResource bibo:volume ?irvolume }
+            OPTIONAL { ?infoResource bibo:pageStart ?irstartPage }
+            OPTIONAL { ?infoResource bibo:pageEnd ?irendPage }
+            OPTIONAL { ?infoResource bibo:doi ?irdoi }
+            OPTIONAL { ?infoResource bibo:pmid ?irpmid }
+            OPTIONAL { ?infoResource bibo:isbn10 ?irisbn10 }
+            OPTIONAL { ?infoResource bibo:isbn13 ?irisbn13 }
+            OPTIONAL { ?infoResource core:placeOfPublication ?irlocale }
             OPTIONAL
             {
                 ?infoResource bibo:reproducedIn ?appearsInObj .
-                ?appearsInObj rdfs:label ?appearsIn .
+                ?appearsInObj rdfs:label ?irappearsIn .
             }
             OPTIONAL
             {
                 ?infoResource core:publisher ?publisherObj .
-                ?publisherObj rdfs:label ?publisher .
+                ?publisherObj rdfs:label ?irpublisher .
             }
             OPTIONAL
             {
-                ?infoResource &lt;http://purl.obolibrary.org/obo/BFO_0000050&gt; ?partOfObj .
-                ?partOfObj rdfs:label ?partOf .
+                ?infoResource obo:BFO_0000050 ?partOfObj .
+                ?partOfObj rdfs:label ?irpartOf .
             }
             OPTIONAL
             {
                 ?infoResource core:hasPublicationVenue ?publishedIn .
-                ?publishedIn  rdfs:label ?journal .
+                ?publishedIn  rdfs:label ?irjournal .
             }
             OPTIONAL
             {
                 ?infoResource core:dateTimeValue ?dateTimeValue .
-                ?dateTimeValue core:dateTime ?dateTime .
+                ?dateTimeValue core:dateTime ?irdateTime .
             }
-            OPTIONAL
-            {
-                ?infoResource core:relatedBy ?editorship .
+            OPTIONAL {
+              { ?infoResource ?property ?editorship .
                 ?editorship a core:Editorship .
-                ?editorship core:relates ?editorObj .
-                ?editorObj rdfs:label ?editor .
+                ?editorship core:relates ?editor .
+                ?editor a vcard:Kind .
+                ?editor vcard:hasName ?vName .
+                ?vName vcard:familyName ?lastName .
+                OPTIONAL { ?vName vcard:givenName ?firstName . }
+                OPTIONAL { ?vName core:middleName ?middleName . }
+                bind ( COALESCE(?firstName, "") As ?firstName1) .
```

Comment:
The `COALESCE` statements were originally there because `bind` fails if one of the variables is unbound. I added the `IF` statement in the `CONCAT` because the extra spaces around editors sans middles name were bugging me and I don't think SPARQL has a trim function. But you're right, the `COALESCE` statements for the first and middle names are unnecessary now. If we add another `IF` statement for the last name we can get rid of `COALESCE` completely. Any idea on performance between the two?